store/src/java/com/zimbra/cs/mailbox/Mailbox.java

```diff
@@ -6753,7 +6751,9 @@ public Message saveDraft(OperationContext octxt, ParsedMessage pm, int id, Strin
         StagedBlob staged;
         InputStream is = pm.getRawInputStream();
         try {
+            ZimbraLog.store.debug("Storage Manager USed for saving Draft is: %s", sm.getClass().getSimpleName());
```

Comment:
typo in the message change `USed` to `used`

Reply:
done